src/server/serverElectron.ts

```diff
@@ -5,4 +5,11 @@
 import ServerBase from './serverBase';
 
 const SERVER_PORT = 8081;
-(new ServerBase(SERVER_PORT)).init();
+// try to read from environment variable to check if the users have set up a specific port to use
+try {
+    const customPort = parseInt(process.env.AZURE_IOT_EXPLORER_PORT); // tslint:disable-line:radix
+    (new ServerBase(customPort || SERVER_PORT)).init();
+}
+catch {
```

Comment:
Might be more clear to catch the specific error(s)/exception that would constitute another attempt at deploying the server rather than any exception.

Reply:
yeah true. We would need to revisit this approach (thinking when a related issue got raised maybe) for future improvement. Kinda want to get rid of manual step as a whole.